src/Fixer/Phpdoc/PhpdocNullableTypeDeclarationFixer.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Phpdoc;
+
+use PhpCsFixer\AbstractPhpdocTypesFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\Fixer\ConfigurableFixerTrait;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Preg;
+
+/**
+ * @phpstan-type _AutogeneratedInputConfiguration array{
+ *  syntax?: 'question_mark'|'union',
+ * }
+ * @phpstan-type _AutogeneratedComputedConfiguration array{
+ *  syntax: 'question_mark'|'union',
+ * }
+ *
+ * @implements ConfigurableFixerInterface<_AutogeneratedInputConfiguration, _AutogeneratedComputedConfiguration>
+ *
+ * @no-named-arguments Parameter names are not covered by the backward compatibility promise.
+ */
+final class PhpdocNullableTypeDeclarationFixer extends AbstractPhpdocTypesFixer implements ConfigurableFixerInterface
```

Comment:
priority needs to be explicitly declared, as there is conflict with other rule

ref https://github.com/swe-productivity/PHP-CS-Fixer/pull/8#discussion_r2596269856